DESCRIPTION

```diff
@@ -1,20 +1,24 @@
 Package: sweary
 Type: Package
 Title: Database of Swear Words in R
-Version: 0.0.0.9000
+Version: 0.0.0.9001
```

Comment:
I wouldn't change development versions. It moves pretty fast. I would keep this 0.0.0.9000 until ve decide on releasing something like v0.1.